src/primitive/identity_gate.jl

```diff
@@ -0,0 +1,20 @@
+export TrivilGate, IdentityGate, igate
+
+abstract type TrivilGate{N} <: PrimitiveBlock{N} end
```

Comment:
Is there other trivial gate? if not let's just keep things simple, use `IdentityGate` directly

Reply:
Yes, we have more in YaoExtensions, like `EchoBlock`.